base/markdown/Julia/Julia.jl

```diff
@@ -7,7 +7,7 @@ We start by borrowing GitHub's `fencedcode` extension – more to follow.
 
 include("interp.jl")
 
-@flavor julia [blocktex, blockinterp, hashheader, list, indentcode, fencedcode,
+@flavor julia [github_table, blocktex, blockinterp, hashheader, list, indentcode, fencedcode,
```

Comment:
I don't think tables will cause conflicts with other block elements, but performance-wise it's probably best to run it as late as possible. For that reason I'd put it just before `paragraph`.